### ISSUE TITLE
Can't scroll using mouse in a terminal ?

### ISSUE THREAD

**Issue by Adham-A** [Canop/bacon#153]
Hello,
I can't seem to scroll using my mouse in a terminal, I tried outside and inside vscode but without success. Is it even possible or is it an issue on my computer ?

**Comment by Canop** [comment]
vscode ? Is that issue related to bacon ?

If it is, then yes, you should be able to use the mouse wheel to scroll the report in bacon.

**Comment by Adham-A** [comment]
### On Linux  
- Scrolling fires the "Up" & "Down" events, same as using those buttons on the keyboard.

### On Windows 
- No events are fired, but there is supposed to be.

Investigations :
I tried getting Mouse Events instead of key events inside the project with no luck, I then tried in an empty project with only `termimad` and I didn't succeed either. I then directly tried with crossterm and it worked, firing `ScrollDown` & `ScrollUp`

After some investigation you have to :

```rust
   // Have to do this part because of crossterm
    let mut stdout = io::stdout();
    execute!(
        stdout,
        EnableMouseCapture, // termimad::crossterm::event::EnableMouseCapture
    )?;
    // Have to do this part because of crossterm`
```

**Comment by Canop** [comment]
Thanks for the investigation. I should have rememembered this myself as I had the problem before in broot.

Regarding your code, I'll comment on the PR

**Comment by Canop** [comment]
Now fixed in 2.13.0

**Comment by willhansen** [comment]
(On linux)

Works in normal terminals, but not in tmux.  I think tmux detects if the program captures mouse events, and forwards the mouse events to the program only if so.  

Probably related to the mentioned `EnableMouseCapture` thing.
